lib/nitra/workers/rspec.rb

```diff
@@ -51,8 +53,22 @@ def run_file(filename, preloading = false)
       }
     end
 
-    def runner_for(filename)
-      args = ["-f", "p", filename]
+    def runner_for(filename, preloading)
+      args = ['-f', 'p']
+
+      if configuration.rspec_formatter && !preloading
+        args << '-f'
+        args << configuration.rspec_formatter
+      end
+
+      if configuration.rspec_out && !preloading
+        args << '--out'
+
+        args << Pathname(configuration.rspec_out).sub_ext("_#{Pathname(filename).sub_ext('')}#{Pathname(configuration.rspec_out).extname}").to_s
```

Comment:
This is a bit terse, perhaps extracting this out with a meaningful method name could be a small win for new peeps.

e.g. `unique_output_file`

what do you think?